File: kotlin/src/test/java/com/example/TestRunner.java
package com.example;

public class TestRunner {
    public static void main(String[] args) {
        StringUtilsTest test = new StringUtilsTest();
        try {
            test.testReverse();
            test.testIsPalindrome();
            test.testCountWords();
            System.out.println("All tests passed!");
        } catch (Throwable e) {
            System.out.println("Test failed: " + e.getMessage());
            System.exit(1);
        }
    }
} 

File: kotlin/src/test/kotlin/com/example/TestRunner.kt
package com.example

object TestRunner {
    @JvmStatic
    fun main(args: Array<String>) {
        val test = StringUtilsTest()
        try {
            test.testReverse()
            test.testIsPalindrome()
            test.testCountWords()
            println("All tests passed!")
        } catch (e: Throwable) {
            println("Test failed: ${e.message}")
            System.exit(1)
        }
    }
} 